Sources/KVKCalendar/DayView.swift

```diff
@@ -138,9 +138,7 @@ extension DayView: TimelineDelegate {
         components.hour = hour
         components.minute = minute
         let date = style.calendar.date(from: components)
-
-        guard let delegate else { return event }
-        return delegate.willAddNewEvent(event, date)
+        return delegate?.willAddNewEvent(event, date) ?? event
```

Comment:
This implementation is problematic. If the user implements the proxy method and returns `nil`, it means that the user does **not want to add** this new event.

Your current writing will result in the use of `event` instead of `nil`, which means that the event will be added.

Reply:
If the user implements a proxy, then the content returned by the proxy method should be strictly used instead of using `event` as the default value. So it is necessary to check in advance whether `delegate` is implemented.